test/test_trixi.jl

```diff
@@ -27,3 +27,51 @@ function test_trixi_run(parameters_file; l2=nothing, linf=nothing, atol=200*eps(
     end
   end
 end
+
+
+
+walkexpr(f, expr::Expr) = f(Expr(expr.head, (walkexpr(f, arg) for arg in expr.args)...))
+walkexpr(f, x) = f(x)
+
+function mapexpr(expr; kwargs...)
+  walkexpr(expr) do x
+    if x isa Expr
+      for (key,val) in kwargs
+        if x.head === Symbol("=") && x.args[1] === Symbol(key)
+          x.args[2] = :( $val )
+          # dump(x)
+        end
+      end
+    end
+    return x
+  end
+end
+
+function test_trixi_include(parameters_file; l2=nothing, linf=nothing,
```

Comment:
Yeah, that's on the TODO list. Since it behaves very similar to `Base.include`, this name seems to be fine for me. As long as we don't extend `Base.include` but call it `Trixi.include`, we do not commit type piracy or something similar.

Reply:
OK, sounds reasonable.